.github/workflows/check-pr-label.yaml

```diff
@@ -0,0 +1,20 @@
+name: PR Label Checker
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - labeled
+      - unlabeled
+
+jobs:
+
+  check_labels:
+    name: Check PR labels
+    runs-on: ubuntu-latest
+    steps:
+      - uses: docker://agilepathway/pull-request-label-checker:latest
+        with:
+          any_of: documentation,enhancement,bug,cicd,test,breaking-change,Feature,scripts
```

Comment:
since all are in lowercase maybe "Feature" should also be lower case?

Reply:
Corrected.